reana/cli.py

```diff
@@ -42,7 +45,7 @@
     'reana-workflow-engine-yadage',
     'reana-workflow-monitor',
     'reana.io',
-]
+] + REPO_LIST_DEMO
```

Comment:
BTW one issue is that some of the demos are not runnable yet (e.g. lhcb one).  Also, other demos may require huuge memory (e.g. the alice one).   So if people would run default `make example`, they would see the command failing.... What about using `REPO_LIST_DEMO` only for the repositories which are known to work well locally?  That's what I meant by referring to "all our working examples"... Mostly just r-d-helloworld, roofit, worldpopulation, recast, bsm... and the others sitting under ALL still until we make them runnable.

Reply:
Running all examples currently gives:

```console
[reana-demo-bsm-search] reana-client create -f reana-cwl.yaml -n bsm-search.cwl
Usage: reana-client create [OPTIONS]
Try "reana-client create --help" for help.

Error: Invalid value for "-f" / "--file": Path "reana-cwl.yaml" does not exist.
[reana-demo-bsm-search] Command 'reana-client create -f reana-cwl.yaml -n bsm-search.cwl' returned non-zero exit status 2.
make: *** [Makefile:99: example] Error 2
```

This is expected because not all demo examples have all engines, e.g. BSM is only Yadage, etc.

Can you please add some logic `if file-exists reana-cwl.yaml then run-given-example-with-cwl else silently-skip-this-workflow-engine fi` to `reana-dev`?

Reply:
Making change and pushing it to your branch...

```diff
@@ -1306,6 +1314,9 @@ def run_example(component, workflow_engine,
     for component in components:
         for workflow_engine in workflow_engines:
             workflow_name = construct_workflow_name(component, workflow_engine)
+            # check whether example contains recipe for given engine:
+            if not os.path.exists(reana_yaml[workflow_engine]):
+                continue
             # create workflow:
             for cmd in [
                 'reana-client create -f {0} -n {1}'.format(
```